src/PosthogAnalytics.ts

```diff
@@ -18,6 +18,8 @@ import posthog, { PostHog } from 'posthog-js';
 import PlatformPeg from './PlatformPeg';
 import SdkConfig from './SdkConfig';
 import SettingsStore from './settings/SettingsStore';
+import { MatrixClientPeg } from "./MatrixClientPeg";
+import { MatrixClient } from "../../matrix-js-sdk";
```

Comment:
This is probably what is causing CI build failures:
```suggestion
import { MatrixClient } from "matrix-js-sdk/src";
```